### ISSUE TITLE
New answers are not treated as new activity on question list

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#357]
https://meta.codidact.com/posts/280018

New activity in the form of questions and edits is being reported correctly on the question list, but new answers are not, nor do they bump the post:

![annotated screenshot](https://user-images.githubusercontent.com/5557942/102703598-cd9dab80-423e-11eb-880a-c50ce1aef367.png)

See the meta report for links to the specific questions.

I'm marking this high priority because of the missing bump.  If posts were being bumped and the activity were merely being misreported, that would be a medium, but not bumping for new answers means they won't get the attention and review they should get.

**Comment by manassehkatz** [comment]
I have also seen other times *but never quite figured it out*, where the bump happened but no visible activity. Is there a list somewhere of what actions are *supposed* to change the "last activity" time?

**Comment by cellio** [comment]
Update: I answered the one marked as "Mithical answered 4 hours ago" and my answer didn't bump.  I just edited that answer and that *did* bump it.  The bump (and time update) seems to be missing specifically for new answers.  This is new behavior.

@manassehkatz everything that changes the state of a post page is supposed to bump: new post, new answer, edit, close, reopen, delete, undelete (is that all of them?).  There is a bug where sometimes activity doesn't show up on the post page itself -- you'll see "last activity 3 hours ago" and everything visible on the post page is weeks old.  One cause of that is a bug with suggested edits.  I'm not sure what else is going on there, but that's a different problem from this one.
